eng/ApiCompat/ApiCompat.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="VerifyCompat">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <RunApiCompatForSrc>true</RunApiCompatForSrc>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.DotNet.ApiCompat" />
+        <PackageReference Include="Azure.Security.KeyVault.Secrets" />
```

Comment:
We would have to reference all the packages we want to be able to validate here.